dotcom-rendering/src/lib/useSignedInStatus.ts

```diff
@@ -0,0 +1,43 @@
+import { getCookie } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+
+export type SignedInStatus = 'Pending' | 'NotSignedIn' | 'SignedIn';
```

Comment:
would it be helpful if the identity-auth library returned a SignedInStatus instead of a boolean/`authState`?

Reply:
For future reference - since the identity-auth library also needs to be used with other applications which haven't been migrated yet, we'll make a decision on this in the future once we've been able to start on that work and assess the various use cases.